src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -336,3 +336,11 @@ hub2.run(currentHub => {
 When you include and configure Sentry, our JavaScript SDK automatically attaches global handlers to _capture_ uncaught exceptions and unhandled promise rejections. You can disable this default behavior by changing the `onunhandledrejection` option to `false` in your GlobalHandlers integration and manually hook into each event handler, then call `Sentry.captureException` or `Sentry.captureMessage` directly.
 
 You may also need to manage your configuration if you are using a third-party library to implement promises. Also, remember that browsers often implement security measures that can block error reporting when serving script files from different origins.
+
+## Why am I seeing events with "Non-Error exception (or promise rejection) captured with keys: ..."
```

Comment:
This is a bit long and we generally do shorter headings in the docs that aren't phrased as questions. Also the body text is searchable so folks will be able to find the content. Does shortening it like so work?

```suggestion
## Events With "Non-Error Exception"
```